### ISSUE TITLE
Date picker is in English (on French side)

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#4314]
### Description

When on the French version of the add experience page, the provided date picker shows all the months in English.

**Environment:** UA

**Comment by gobyrne** [comment]
We already have a backlog item for updating the date picker (at least for the experience pages). I think that's the only applicant-facing place that we have date-pickers (I'll double check).

But that means I'm leaning for #wontfix? or is that actually both?

**Comment by petertgiles** [comment]
Basically, we're using the browser's date picker so we don't get to control what language it is in or really, anything about it.  See how different it is between Firefox and Chromium on my machine.  The colour and date format aren't even the same.
![image](https://user-images.githubusercontent.com/8978655/199079570-e1653151-8388-48f7-aeaa-57070d9bade0.png)
![image](https://user-images.githubusercontent.com/8978655/199079629-836dcc1c-d96d-45de-aa22-17854f026d88.png)

If we want to stick with the browser datepicker than this is #wontfix (or #cantfix I think :laughing:).  If we do want to swap in a new datepicker then that really has to happen before we look at this.

**Comment by esizer** [comment]
We have a new date input that does properly update to French. Should be safe to close 👍 
![Screenshot 2023-06-24 135246](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/e5d7342e-ea5c-4292-9c25-23093d20ee11)
